src/style.css

```diff
@@ -1,3 +1,61 @@
 body {
   margin: 0;
+  width: 100%;
 }
+
+* {
+  --color-line: #00acdc;
```

Comment:
It is not recommended to set variables on the universal selector. Variables should be declared on :root or body for better performance.